examples/ex-07.c

```diff
@@ -0,0 +1,772 @@
+/*BHEADER**********************************************************************
+ * Copyright (c) 2013, Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory. Written by
+ * Jacob Schroder, Rob Falgout, Tzanio Kolev, Ulrike Yang, Veselin
+ * Dobrev, et al. LLNL-CODE-660355. All rights reserved.
+ *
+ * This file is part of XBraid. For support, post issues to the XBraid Github
+ * page.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License (as published by the Free
+ * Software Foundation) version 2.1 dated February 1999.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the IMPLIED WARRANTY OF MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the terms and conditions of the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ ***********************************************************************EHEADER*/
+
+/**
+ * Example:       ex-07.c
+ *
+ * Interface:     C
+ *
+ * Requires:      only C-language support
+ *
+ * Compile with:  make ex-07
+ *
+ * Help with:     ./ex-07 -help
+ *
+ * Sample run:    mpirun -np 2 ex-07 -rank 3
+ *
+ * Description:   Solve the chaotic Lorenz system,
+ *
+ *                  {x' = sigma (y - x),    x(0) = -1.8430428,
+ *                  {y' = x (rho - z) - y,  y(0) = -0.07036326,
+ *                  {z' = xy - beta z,      z(0) = 23.15614636.
+ *
+ *                with sigma=10, beta=8/3, and rho=28,
+ *                and estimate the backward Lyapunov vectors along the trajectory
+ *                parallel-in-time using MGRIT equipped with Delta correction.
+ *                The initial condition should be a point close to the strange attractor.
+ *
+ *                TODO: Give some high-level info about the algorithm.
```

Comment:
I would give a single high-level description of the algorithm in an .md file in /docs.  Then just reference that here.